rails/spec/controllers/api/v1/report_learners_es_controller_spec.rb

```diff
@@ -271,4 +274,31 @@
     end
   end
 
+  describe "jwt query" do
```

Comment:
A spec test for the JWT learners route should be added too, if it isn't too hard. I'm not sure how a spec will work with elastic search, though.

I think it is important to verify that un authorized users can't send queries in order to access learners.
Perhaps there is a way to mock ES and still check this authorization code.

Reply:
Added new tests.